.husky/pre-commit

```diff
@@ -1,6 +1,15 @@
-#!/bin/sh
+#!/usr/bin/env bash
+set -eo pipefail
 [ -n "$CI" ] && exit 0
 
 . "$(dirname "$0")/_/husky.sh"
 
 yarn lint-staged
```

Comment:
@gjedlicska - is `yarn lint-staged` now duplicated by `eslint` in pre-commit?  If so, I can make this run only if the `pre-commit` command is not found.